packages/docs/static/definitions/schemas/0.0.1/KYCAccInvAttestation

```diff
@@ -0,0 +1,23 @@
+{
+  "$ref": "#/definitions/KYCAccInvAttestation",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "KYCAccInvAttestation": {
```

Comment:
"KYC" as outlined by the Bank Secrecy Act is a set of requirements that are generally considered separate from the affirmative accreditation verification requirements described in Reg D 506 (c).  This means that it's possible to perform an affirmative accreditation verification without performing a full KYC check (and vice versa).  Would this credential only be available for individuals who have gone through both a KYC process _and_ an affirmative accreditation check?

Reply:
will rename to indiv/entity rather than kyc/kyb

Reply:
but good catch-- definitely the idea is for these to be composable and for issuers of accinv to have NOT NECESSARILY done both; both should be "account-bound" and IDV'd but not necessarily to a KYC level if only issuing AccInv VC

Reply:
addressed in latest commit